### ISSUE TITLE
Intro in documentation needs update

### ISSUE THREAD

**Issue by jblotus** [doctrine/migrations#1030]
The documentation for 3.0 has not been updated - I was going to create a pull request to fix it but it references a phar file for v2.
Is there a phar file for version 3? Is this still a recommended way to use this package?

https://github.com/doctrine/migrations/blob/3.0.1/docs/en/reference/introduction.rst

**Comment by goetas** [comment]
@jblotus you are right, somehow that part was forgotten in the update process... it would be great if you could update it!

Regarding the PHAR install, it seems that they are currently not updated.

@greg0ire @SenseException @alcaeus Does some of you remember how were the PHAR uploaded to github? Is that something that should be done by the @doctrinebot  ? (the phar generation is tested and works, it seems that is just not being uploaded)

**Comment by alcaeus** [comment]
IIRC it was always done manually. The release bot does not package phar files and add them as artefact.

**Comment by SenseException** [comment]
It would be useful if doctrinebot could handle the release of the PHAR for Migrations.
